e3nn/networks.py

```diff
@@ -27,9 +31,16 @@ def __init__(self, Rs_in, Rs_hidden, Rs_out, lmax, layers=3, max_radius=1.0, num
 
         K = partial(Kernel, RadialModel=RadialModel, get_l_filters=partial(o3.selection_rule, lmax=lmax))
 
-        def make_layer(Rs_in, Rs_out):
+        def make_layer(Rs_in, Rs_out, feature_product=feature_product):
```

Comment:
```suggestion
        def make_layer(Rs_in, Rs_out):
```
no need to pass this argument